core/tx_pool.go

```diff
@@ -496,13 +497,29 @@ func (pool *TxPool) Content() (map[common.Address]types.Transactions, map[common
 // Pending retrieves all currently processable transactions, grouped by origin
 // account and sorted by nonce. The returned transaction set is a copy and can be
 // freely modified by calling code.
-func (pool *TxPool) Pending() (map[common.Address]types.Transactions, error) {
+//
+// The enforceTips parameter can be used to do an extra filtering on the pending
+// transactions and only return those whose **effective** tip in the current
```

Comment:
Last part of this sentence got lost?

Reply:
Yeah, that's what happens when someone interrupts me and I forgot what I was doin g:))